doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -2032,36 +2078,39 @@ Response:
         [
           {
             "id": "abcdefg",
-            "subject": {
+            "description": "Example Context Source",
+            "dataProvided": {
               "entities": [
                 {
                   "id": "Bcn_Welt",
                   "type": "Room"
                 }
               ],
-              "attributes": [
+              "attrs": [
                 "temperature"
               ]
             },
-            "callback": "http://weather.example.com/ngsi",
-            "metadata": {
-              "providingService": {
-                 "value": "weather.example.com",
-                 "type": "Text"
-               },
-              "providingAuthority": {
-                 "value": "AEMET - Spain",
-                 "type": "Text"
-               }
+            "provider": {
+              "http": {
+                "url": "http://contextsource.example.org/v2/entities"
+              },
+              "legacy": false,
```

Comment:
Taking into account what I said at https://github.com/telefonicaid/fiware-orion/pull/2990/files#r144080050, `"legacy": ...` should be removed in all examples.

Reply:
fixed 78ce71dfa1b2c6952018ede3a9ef640c65abc1dd